apps/api/tests/test_guest.py

```diff
@@ -105,7 +107,9 @@ def test_successful_guest_verification_provides_identity(
 
 
 @patch("auth.guest_auth._get_existing_key", autospec=True)
+@patch("auth.guest_auth._remove_guest_key", autospec=True)
 def test_invalid_guest_verification_is_unauthorized(
+    mock_remove_guest_key: AsyncMock,
```

Comment:
Suggestion [repeating]: the mock is currently unused in the function. We could call the `assert_awaited_once_with` method to further validate its usage.